prow/plugins/label/label.go

```diff
@@ -96,14 +121,15 @@ func getLabelsFromREMatches(matches [][]string) (labels []string) {
 
 // getLabelsFromGenericMatches returns label matches with extra labels if those
 // have been configured in the plugin config.
-func getLabelsFromGenericMatches(matches [][]string, additionalLabels []string) []string {
+func getLabelsFromGenericMatches(matches [][]string, additionalLabels []string, invalidLabels *[]string) []string {
 	if len(additionalLabels) == 0 {
 		return nil
 	}
 	var labels []string
 	for _, match := range matches {
 		parts := strings.Split(match[0], " ")
 		if ((parts[0] != "/label") && (parts[0] != "/remove-label")) || len(parts) != 2 {
+			*invalidLabels = append(*invalidLabels, match[0])
 			continue
 		}
 		for _, l := range additionalLabels {
```

Comment:
We need to add `parts[1]` to `invalidLabels` if it is not one of the `additionalLabels`.

Reply:
I think I am confused: I believe that parts[0] is a subset of match[0], because:
```go
parts := strings.Split(match[0], " ")
```
Did I miss your point?